app/javascript/entrypoints/pdc/maximum_file_upload.es6

```diff
@@ -0,0 +1,24 @@
+export default class MaximumFileUpload {
+
+    constructor(upload_id, save_id, error_id = "file-error") {
+      this.upload_element = $("#"+upload_id);
+      this.save_element = document.getElementById(save_id);
+      this.error_element = document.getElementById(error_id)
```

Comment:
A convention that I've used is to distinguish jQuery objects by prefixing with `$`. Here, things are small enough that it's unlikely we'd get confused, but if there's more code, it's possible to forget what kind of "element" we're dealing with. I'll put a little item on the agenda.